### DISCUSSION TITLE
should I put snacks.nvim over previous configs?

### DISCUSSION THREAD

**Discussion by AnirudhG07** [folke/snacks.nvim#213] [General]
I found out about snacks.nvim today and it is very good. But I have lots of stuff preconfigured as well. I am not sure if I add it, will it clash with everything else? Is it a good idea to remove and put this? or can this be put on top of each other and still things work FAST?
Here is my [nvim plugins](https://github.com/AnirudhG07/dotfiles/tree/stow/nvim/.config/nvim/lua/anirudh/plugins), pls tell which is best for me.

**Comment by silndoj** [comment]
im facing your problem right now !
Did u found anything yet?

**Comment by folke** [comment]
Nothing is enabled by default, so you can enable things you want

**Reply by AnirudhG07** [reply]
Thanks so much for clarifying! 
I have a question, many plugins have `nvim-notify` as a requirement, can snacks be their replacement if I remove nvim-notify?

**Reply by folke** [reply]
no plugin that I know requires `nvim-notify`. They use `vim.notify` which snacks notifier also replaces

**Reply by AnirudhG07** [reply]
in my configs, I have `rcdrriga/nvim-notify` dependency for `noice.lua`. But since you only made it, I can just replace it with snacks? will you recommend that?

**Reply by folke** [reply]
noice automatically uses the snacks notifier if it's installed and enabled

**Reply by AnirudhG07** [reply]
great
